tests/unit/Actions.spec.ts

```diff
@@ -7,12 +7,16 @@ import sinon from 'sinon'
 import { axios } from '@/utils'
 import Actions from '@/components/common/Actions.vue'
 import mockRouter from './MockRouter'
+import LegalServices from '@/services/legal.services'
 
 Vue.use(Vuetify)
 
 const vuetify = new Vuetify({})
 const store = getVuexStore()
 
+// mock services function
+const mockUpdateFiling = jest.spyOn((LegalServices as any), 'updateFiling').mockImplementation()
```

Comment:
This is a mock for the `updateFiling` function in Legal Services class.

This also provides a default implementation so it actually "runs".

I just made this a global variable in this test file and "reset" it when I need.